Gems/ScriptAutomation/Code/Source/ScriptAutomationSystemComponent.h

```diff
@@ -9,11 +9,14 @@
 #pragma once
 
 #include <ScriptAutomation/ScriptAutomationBus.h>
+#include <ImageComparisonSettings.h>
 
 #include <AzCore/Component/Component.h>
 #include <AzCore/Component/TickBus.h>
 #include <AzCore/std/smart_ptr/unique_ptr.h>
 
+#include <AzFramework\Asset\AssetCatalogBus.h>
```

Comment:
These need to be forward slashes
```suggestion
#include <AzFramework/Asset/AssetCatalogBus.h>
```